[master] Fails to compile with error: cannot find -lcurses: No such file or directory

```console
# make
cc -Wall -Wextra    ttyplot.c  -lcurses -ltinfo -o ttyplot
/usr/lib/gcc/x86_64-pc-linux-gnu/11/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find -lcurses: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [<builtin>: ttyplot] Error 1

# pkg-config --libs ncurses
-lncurses -ltinfo
  ^
```

Pull request upcoming...

--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@ DESTDIR   ?=
 PREFIX    ?= /usr/local
 MANPREFIX ?= $(PREFIX)/man
 CFLAGS += -Wall -Wextra
-LDLIBS += -lcurses -ltinfo
+LDLIBS += `pkg-config --libs ncurses 2>/dev/null || echo '-lcurses -ltinfo'`
 PKG = ttyplot_1.4-1
 PKGDIR = $(PKG)/usr/local/bin
 

